src/lib/tvgPictureImpl.h

```diff
@@ -90,24 +90,30 @@ struct Picture::Impl
         return ret;
     }
 
-    uint32_t reload()
+    bool reloadPaint()
     {
-        if (loader) {
-            if (!paint) {
-                if (auto p = loader->paint()) {
-                    paint = p.release();
-                    loader->close();
-                    if (w != loader->w || h != loader->h) {
-                        if (!resizing) {
-                            w = loader->w;
-                            h = loader->h;
-                        }
-                        loader->resize(paint, w, h);
-                        resizing = false;
+        if (loader && !paint) {
+            if (auto p = loader->paint()) {
+                paint = p.release();
+                loader->close();
```

Comment:
#1338 forces reload, so reload doesn't seem necessary. I think loader->w, h are already updated. Can't we just update this->w, h?